server/helpers/userHelpers.js

```diff
@@ -1,11 +1,96 @@
+
+/**
+ * @description Removes all white space character, if any
+ *
+ * @method cleanString
+ * @param {Strng} string The string to remove white space(s)
+ * @param {Boolean} removeAll: if false, multiple white space will reduce to single;
+ * @returns {Object} An object with trimed attributes
+ * @method sanitize
+ */
+export const sanitize = (string, removeAll) => {
+  let all = typeof removeAll !== "undefined" ? removeAll : true;
+  if (string) {
+    return all
+      ? string.trim().replace(/[ ]+/g, "") // remove all white spaces
+      : string.trim().replace(/[ ]+/g, " "); // collapse multiple white spaces to single
+  }
+};
+
 /**
  * @description Get the app base url
  *
- * @param {object} httpObj HTTP request object
+ * @param {object} httpRequestObject HTTP request object
  * @returns {string} the base url
  * @methodget BaseUrl
  */
+export const getBaseUrl = httpRequestObject => {
+  return `${httpRequestObject.protocol}://${httpRequestObject.get(
+    "host"
+  )}/api/v1`;
+};
 
-export const getBaseUrl = httpObj => {
-  return `${httpObj.protocol}://${httpObj.get("host")}/api/v1`;
+/**
+ * @description Capitalize the first character of a string to uppercase
+ *
+ * @param {Array} skip array of words to skip during the conversion
+ * @param {String} string the string to convert
+ * @param {boolean} isSentence determine if the string to convert is a single word or a sentence
+ * @returns {string} the converted string
+ * @method generateSentenceCase
+ */
+const generateSentenceCase = (skip, string, isSentence) => {
+  if (string) {
+    let str = string.toString().trim();
+    if (isSentence) {
+      string = string.toString().trim();
+      return skip.includes(string)
+        ? string
+        : sanitize(
+            string
+              .charAt(0)
+              .toUpperCase()
+              .concat(string.substr(1))
+          );
+    }
+    return (str = skip.includes(str)
+      ? str
+      : str
+          .charAt(0)
+          .toUpperCase()
+          .concat(str.substr(1)));
+  }
+};
+
+/**
+ * @description Capitalizes the first character of a string
+ *
+ * @method capitalizeFirstChar
+ * @param {String} string The string to be converted
+ * @param {Boolean} options
+ * Note:
+ * - options.recursive, set to true to capitalize the first character of each word in the string
+ * or false to capitalize only the first character of the string
+ * - options.skip: array of strings/words to skip
+ * - options.seperator the seperator to use when splitting the string
+ * @returns {String} The converted string
+ */
+export const capitalizeFirstChar = (string, options = {}) => {
```

Comment:
This function is doing to much for something that just capitalizes the first character. If you can't simplify this method please leave it out